style.css

```diff
@@ -0,0 +1,84 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  background-image:
+    url(images/ellipse_3.png),
+    url(images/Ellipse_4.png),
+    url(images/Ellipse_6.png),
+    url(images/ellipse_11.png),
+    url(images/Rectangle_4.png),
+    url(images/Rectangle_6.png),
+    url(images/Rectangle_5.png),
+    url(images/Rectangle_7.png),
+    url(images/ellipse_12.png),
+    url(images/ellipse_5.png),
+    url(images/ellipse_7.png),
+    url(images/Ellipse_13.png);
+  background-position-x: 128%, 57%, 6%, -6%, 104%, -11%, 50%, 27%, 71%, 30%, 65%, 68%;
+  background-position-y: 22%, 87%, 95%, 20%, 16%, 18%, 90%, 20%, 70%, 21%, 16%, 72%;
+  background-repeat: no-repeat;
```

Comment:
- [ ] **As suggested by the previous code reviewer**, I think it would be better to adjust your background image to match that of the Figma design, You may try to use the following 👇  to fix the ```background-image``` issue 😃😃
![header-shapes mobile](https://user-images.githubusercontent.com/102757126/212360502-04127925-0750-42de-824b-49b190092804.png)